docs/serving/samples/tag-header-based-routing/README.md

```diff
@@ -0,0 +1,118 @@
+This sample illustrates the feature "Tag Header Based Routing".
+
+## Tag Header Based Routing Feature
+
+The feature "Tag Header Based Routing" allows users to send requests directly to specific tagged Revisions with
+the same URL of Knative Service. In order to achieve this, users only need to set the specific header "Knative-Serving-Tag:
+{revision-tag}" into the request.
+
+Currently Istio, Contour and Kourier Ingress support this feature.
+
+## Prerequestie
+
+1. A Knative cluster that has an Istio Ingress controller or Contour Ingress controller installed
+with Knative version 0.16 and above.
+
+1. Move into the docs directory:
+
+```shell
+cd $GOPATH/src/github.com/knative/docs
+```
```

Comment:
Why is this step required? Doesn't seem relevant?

Reply:
Because in the step of "Setting up the revisions with tag", we ask users to run command 
```
kubectl apply -f docs/serving/samples/tag-header-based-routing/sample.yaml
```

The path `docs/serving/samples/tag-header-based-routing/sample.yaml` is the relative directory which requires users move into $GOPATH/src/github.com/knative/docs